code/modules/clothing/masks/gasmask.dm

```diff
@@ -130,23 +121,21 @@
 	T.change()
 
 /obj/item/clothing/mask/gas/voice/proc/change()
-
-	var/obj/item/clothing/mask/A
-	A = input("Select Form to change it to", "BOOYEA", A) as null|anything in clothing_choices
-	if(!A ||(usr.stat))
+	var/choice = input(usr, "Select Form to change it to", "BOOYEA") as null|anything in clothing_choices
+	if(!choice || !usr.Adjacent(src) || usr.incapacitated())
 		return
-
-	desc = null
-	permeability_coefficient = 0.90
-
-	desc = A.desc
-	name = A.name
-	flags = A.flags
-	icon = A.icon
-	icon_state = A.icon_state
-	item_state = A.item_state
-	can_flip = A.can_flip
-	body_parts_covered = A.body_parts_covered
+	var/obj/item/clothing/mask/chosen_type = clothing_choices[choice]
```

Comment:
You may want to explain what's going on here in a comment, since it's undocumented behavior